daily-test/variables.yaml

```diff
@@ -0,0 +1,33 @@
+project:
+  id: 'eq-daily-performance-test'
+  folder_id: ''
+  billing_account: ''
+  region: 'europe-west2'
+  label: 'sandbox-eq'
+
+k8s:
+  node_machine_type: 'n1-standard-1'
+  timeout: '60m'
+  available_cpu_per_pod: '0.7'
```

Comment:
`per_node` instead of `per_pod`?

Reply:
`per_node` makes more sense, `per_pod` assumes one pod per node.

Reply:
The env var used in the task file is `REQUESTED_CPU_PER_POD`. This makes more sense in the task file as it lives with the implementation of the task, however, I'm in two minds of whether to change it there to `REQUESTED_CPU_PER_NODE` as well. Thoughts?

Reply:
@ajmaddaford spoke and think we said 'available' was fine with the comment. I think `available, `requested` or `allocatable` is fine?